src/terminal.go

```diff
@@ -857,6 +859,8 @@ func (t *Terminal) environ() []string {
 	env = append(env, fmt.Sprintf("FZF_LINES=%d", t.areaLines))
 	env = append(env, fmt.Sprintf("FZF_COLUMNS=%d", t.areaColumns))
 	env = append(env, fmt.Sprintf("FZF_POS=%d", util.Min(t.merger.Length(), t.cy+1)))
+	env = append(env, fmt.Sprintf("FZF_HEADERCLICK_LINE=%d", t.headerClickLine))
+	env = append(env, fmt.Sprintf("FZF_HEADERCLICK_COLUMN=%d", t.headerClickColumn))
```

Comment:
I'd like to avoid exporting these on other events. To do so, we can reset the variables to zero after running the bound actions, and export the variables when they are greater than zero,

Reply:
Would this be a suitable way to handle the reset?

```
if my >= 0 && my < numLines && mx >= 0 {
    t.clickHeaderLine = my + 1
    t.clickHeaderColumn = mx + 1
    evt := tui.ClickHeader
    res := doActions(actionsFor(evt))
    t.clickHeaderLine = 0
    t.clickHeaderColumn = 0
    return res
}
```

Reply:
Yes, and,

```go
if t.headerClickLine > 0 {
  env = append(...)
  env = append(...)
}

Reply:
See 354b8b26dc310aaefc770c6a415e9be45a61ff9d